janitor/functions/pivot.py

```diff
@@ -371,115 +257,88 @@ def _data_checks_pivot_longer(
 
     if index is not None:
         if is_list_like(index) and (not isinstance(index, tuple)):
-            index = list(index)
+            index = [*index]
```

Comment:
Do you mind educating me on the difference between these?
`list(index)` and `[*index]`; and I believe there is also the option of `index[:]` to copy; I've just always relied on the first option..
Why the decision to use one over the other now?

Reply:
No reason ... There is microperformance, but it is useless here. I just like the unpacking